web/src/reducers/activities.selectors.js

```diff
@@ -30,7 +30,7 @@ const selectBudgetForActivity = ({ budget }, { aKey }) =>
 
 export const makeSelectCostAllocateFFP = () =>
   createSelector(
-    [selectActivityByKey, selectBudgetForActivity],
+    [selectActivityByIndex, selectBudgetForActivity],
```

Comment:
I thought this might help with the above failing test scenario, but no dice. @mgwalker, if you can spare some time Tuesday, I would so appreciate a pair!